app/src/androidTest/java/com/jnj/vaccinetracker/syncexecutionscript/DraftParticipantScript.kt

```diff
@@ -86,7 +87,8 @@ class DraftParticipantScript {
         val pictureResourceName: String? = "test_script_image.jpeg"
         val image = pictureResourceName?.let { readPicture(it) }
         val telephone: String? = null
-        val yearOfBirth = 1994
+        val birthDate = DateTime(1999, 6, 6)
+        val isBirthDateAnApproximation = false
```

Comment:
How about we change the naming to **_isBirthDateEstimated_**

Reply:
I agree, the 'Estimated' is the word used in OMRS in this case

Reply:
Ok, I've changed that everywhere.